xc7/utils/prjxray_routing_import.py

```diff
@@ -694,6 +700,11 @@ def main():
                 import_graph_edges(conn, graph, node_mapping)
             )
 
+        print('{} Writing node map.'.format(now()))
+        with open(args.write_rr_node_map, 'wb') as f:
+            pickle.dump(node_mapping, f)
```

Comment:
Is using a pickle verse a json file more performant?

Reply:
Pretty much.  This is a diagnostic leaf, I wanted a low impact both in runtime and disk space.